python-avd/pyavd/_eos_designs/schema/schema_fragments/validation_profiles.schema.yml

```
@@ -0,0 +1,75 @@
+# Copyright (c) 2025 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  validation_profiles:
+    type: list
+    primary_key: name
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+        parent_profile:
+          type: str
+          description: |-
+            Inherit settings from a parent profile defined under `validation_profiles`.
+            Max one level of profile inheritance.
```

Let's align the description like in other places.

```suggestion
            Max one level of profile inheritance: profile -> parent_profile
```